packages/swarm-docs/src/pages/index.jsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react'
+import React from 'react';
 
-import Layout from '../components/layout'
+import Layout from '../components/layout';
 
 const IndexPage = () => (
   <Layout>
-    <h1 className="text--display">Hello</h1>
-    <p className="text--body">Meetup's System in testing for travis</p>
+    <h1 className="text--display" style={{fontWeight: 700}}>Hello</h1>
```

Comment:
will there be a dedicated page to demo the different font weights at some point? Is that something we could add now? I can imagine it would be quite useful to have a reference for what each one looks like next the others - maybe examples of h1, h2, h3, and p in each weight? @hsbacot is there a typography page that exists or is planned to show this kind of thing?

Reply:
I had the same question in the main ticket: https://meetup.atlassian.net/browse/SDS-930?focusedCommentId=107869&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-107869

@hsbacot, let me know if adding a page is what you have in mind, also which fonts - I don't believe I have full context on why we have 4 fonts now instead of 3 and if all of them are supposed to be used.

Reply:
In any case, a docs page is probably a separate ticket so I'll merge this one.

Reply:
`/typography` exists in the docs and will be receiving an update soon. Design is working on how the docs page will look.